tests/drivers/retained_mem/api/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,22 @@
+/ {
+	cpuapp_ram0_ret@22004000 {
```

Comment:
I'm wondering if this should be a subnode of `&cpuapp_ram0`, but it would probably require greater refactoring of the memory nodes.
Maybe a subnode of `/reserved-memory` then. Someone who knows the device tree specification better could chip in.